notebooks/2. Data fusion.ipynb

```diff
@@ -117,23 +119,37 @@
    "metadata": {},
    "source": [
     "### [Alternative] Building fuser using BGT data\n",
-    "First, download the required BGT data. E.g. for our demo point cloud, run the following lines in a terminal: \n",
-    "```sh\n",
-    "$ cd ../scripts\n",
-    "$ python bgt_building_scraper.py --bbox_region \"119300,485150,119350,485100\" --out_folder ../datasets/bgt/\n",
-    "```"
+    "First, download and parse the required BGT data. E.g. for our demo point cloud, run the following cell: "
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "caring-bryan",
+   "id": "aquatic-gauge",
    "metadata": {},
    "outputs": [],
    "source": [
     "# Data folder for the BGT fuser.\n",
     "bgt_data_folder = '../datasets/bgt/'\n",
     "\n",
+    "# Specify the bounding box of the area we want to scrape.\n",
+    "# For example, we can find the bounding box of a LAS file.\n",
+    "bbox_padded = las_utils.get_bbox_from_las_folder('../datasets/pointcloud/', padding=10)\n",
+    "\n",
+    "# Scrape data from the Amsterdam WFS, this will return a json response.\n",
+    "json_response = bgt_scraper.scrape_amsterdam_bgt('BGT_PND_pand', bbox=bbox_padded)\n",
+    "\n",
+    "# Parse the downloaded json response.\n",
+    "bgt_scraper.parse_buildings(json_response, out_folder=bgt_data_folder)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "caring-bryan",
+   "metadata": {},
+   "outputs": [],
+   "source": [
     "# Building fuser using BGT building footprint data.\n",
     "bgt_building_fuser = fusion.BGTBuildingFuser(Labels.BUILDING, building_offset=1, bgt_folder=bgt_data_folder)"
```

Comment:
Veiliger om een file mee te geven dan een hele folder, voor wanneer er verschillende datasets in staan:
```
bgt_file = bgt_folder + 'bgt_buildings.csv'
bgt_building_fuser = fusion.BGTBuildingFuser(Labels.BUILDING, building_offset=1, bgt_file=bgt_file)
```